netsim/devices/cumulus_nvue.yml

```diff
@@ -23,14 +23,18 @@ features:
     activate_af: True
   ospf:
     unnumbered: True
+  vlan:
+    model: switch
+    svi_interface_name: "vlan{vlan}"
+    subif_name: "{ifname}.{vlan.access_id}"
   vrf: True
 clab:
   mtu: 1500
-  kmods:
+  kmods: [ ebtables ]
```

Comment:
I don't think this does what you think it will do ;)

Reply:
It didn't, but it will once we fix it: https://github.com/ipspace/netlab/pull/1520